decidim-core/app/cells/decidim/progress_bar_cell.rb

```diff
@@ -45,29 +43,27 @@ def units_name_text
       I18n.t(units_name, count: progress)
     end
 
+    # deprecated
```

Comment:
I'd replace it by `# redesign deprecated`

Reply:
`# REDESIGN_PENDING: deprecated` perhaps? Unless there is a better way to mark methods as deprecated in Rails, some similar to @deprecated in js-doc, where you can be warned about deprecation in your IDE, like:
![imagen](https://user-images.githubusercontent.com/817526/228455926-f85e5711-d82f-4204-b73f-ace51071201f.png)

Reply:
`# REDESIGN_PENDING: deprecated` sounds good to me! 👍🏾